packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPicture.tsx

```diff
@@ -48,16 +60,35 @@ export const _PictureInPictureInPicture = (props: _PictureInPictureInPictureProp
  */
 const PictureInPictureInPictureContainer = (props: {
   primaryView: ReactChild;
-  secondaryView: ReactChild;
+  secondaryView?: ReactChild;
   onClick?: () => void;
+  ariaLabel: string;
 }): JSX.Element => (
-  <div style={tileContainerStyles} onClick={props.onClick}>
+  <aside
+    style={tileContainerStyles}
+    onClick={props.onClick}
+    onKeyPress={(e) => props.onClick && handleKeyDown(e, props.onClick)}
+    aria-label={props.ariaLabel}
+    tabIndex={props.onClick ? 0 : -1} // Only allow focus to be set if there is a click handler
+  >
     {props.primaryView}
     <div style={secondaryTileFloatingStyles}>{props.secondaryView}</div>
-  </div>
+  </aside>
 );
 
+/**
+ * For keyboard navigation - when this component has active focus, enter key and space keys should have the same behavior as mouse click.
+ */
+const handleKeyDown = (e: React.KeyboardEvent<HTMLElement>, onClickCallback: () => void): void => {
+  if (e.key === 'Enter' || e.key === ' ') {
+    // If the PiP is clicked through keyboard, perform the onClick behavior only.
+    e.preventDefault();
```

Comment:
what happens when we don't precent default behavior?

Reply:
Actually this is just a div, so probably nothing, I'll remove it

Reply:
removed